bahmnicore-api/pom.xml

```diff
@@ -6,7 +6,7 @@
         <artifactId>bahmni</artifactId>
         <version>0.94-SNAPSHOT</version>
     </parent>
-    <artifactId>bahmnicore-api</artifactId>
+    <artifactId>bahmnicore-ts-api</artifactId>
```

Comment:
@sivareddytw any specific reason we renamed to **bahmnicore-ts-api** ?

Reply:
@binduak we are publishing this new artefact temporarily in order not to push these changes to Product's artefacts. Will update these artefacts when raising PR against `master` branch.